containers/pre_built_workers/java/run_worker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2020 gRPC authors
```

Comment:
Since this is a new file, 2021 would be appropriate.

Reply:
Updated 👍